templates/posts/index.html

```diff
@@ -19,14 +19,71 @@ <h2>Post title: {{ post.title }}</h2>
                                 <a href="{{host}}/site/posts/deletepost/{{post.id}}"><button type="button" class="btn" onclick="return confirm('Are you sure?');">Delete post</button></a>
                             {% endif %}
                             <a href="{{host}}/site/posts/comments/{{post.id}}"><button type="button" class="btn">Comments</button></a>
+                            <button type="button" class="btn" id="{{post.id}}">Add comment</button>
```

Comment:
you should probably add a more meaningful id name to the button so that it does not conflict with other possible stuff, add-comment-id? or something similar
this will have to be updated below when you are grabbing that element by id